fleet.go

```diff
@@ -12,3 +12,33 @@ type Fleet struct {
 	ArriveIn     int
 	BackIn       int
 }
+
+// Fleet is not returning and we can calculate the in air time in sec. The function returns 0 if the fleet is returning!
+func (f Fleet) InAir() int64 {
+	if !f.ReturnFlight {
+		TotalFleetFlightTime := (f.BackIn - f.ArriveIn)
+		ArriveAt := time.Now().Unix() + f.ArriveIn
+		FleetStartAt := (ArriveAt - TotalFleetFlightTime)
+		InAirForSec := time.Now().Unix() - FleetStartAt
+		return InAirForSec
+	}
+	return 0
+}
+
+// Fleet is not returning and we can calculate the total time the Fleet needs to arrive. The function returns 0 if the fleet is returning!
+func (f Fleet) Flighttime() int64 {
+	if !f.ReturnFlight {
+		return (f.BackIn - f.ArriveIn)
```

Comment:
ArriveIn is the amount of seconds to go to destination.
BackIn is the amount of seconds to go to destination and back home.
If you remove `ArriveIn` from `BackIn`, what's left is the amount of seconds to go back home if you were to recall now.
right ?!